pkg/loki/loki_test.go

```diff
@@ -0,0 +1,135 @@
+//+build !race
```

Comment:
There's a race condition in promtail itself, so I'm just going to not use the race detector here for now